### ISSUE TITLE
After upgrading to Gnome 40 in the dekstop page the dock show some shadow margins around?

### ISSUE THREAD

**Issue by tassosblackg** [vinceliuice/WhiteSur-gtk-theme#404]
Check this issue it's probably because of the theme?

[`ImageIssue`](https://forum.manjaro.org/t/square-borders-on-dock-gnome-40/62839/5?u=tassosblackg)

How I can detect the part of the code causing these margins in order to fix it?

**Comment by vinceliuice** [comment]
Does this theme have this issue too?

**Comment by tassosblackg** [comment]
yes, it's me that reported the issue using WhiteSur theme for the shell. I've also noticed similar shadows with other themes like BigSure or Yaru also. Is it possible an issue with the dash-to-dock extension? Because the default theme or the Orchis doesn't have this problem

**Comment by vinceliuice** [comment]
I think manjaro have done something change to the dash to dock, I'll download the last manjaro gnome for testing and find a fix for it

**Comment by vinceliuice** [comment]
![VirtualBox_Manjaro_21_08_2021_00_03_24](https://user-images.githubusercontent.com/7604295/130261545-0dcfa546-d282-4ff3-9e73-b4f37b28bd5e.png)

**Comment by vinceliuice** [comment]
Just find a padding issue on manjaro, but did not find the shadow issues

**Comment by tassosblackg** [comment]
![image](https://user-images.githubusercontent.com/22836483/130286846-8c209b2e-4699-4f05-9df5-87aaa576d3c8.png)

Manjaro Info 
DISTRIB_ID=ManjaroLinux
DISTRIB_RELEASE=21.1.0
DISTRIB_CODENAME=Pahvo
DISTRIB_DESCRIPTION="Manjaro Linux"

Gnome Version 40.3.0

**Comment by tassosblackg** [comment]
the only solution I found is through dash-to-dock extension's Appearance settings by disabling :
- Use built-in theme
And enable
- force strait corner 
- shrink dock size

**Comment by vinceliuice** [comment]
Installed this version https://github.com/ewlsh/dash-to-dock/tree/ewlsh/gnome-40 instead, manjaro's dash to dock is so bug!

**Comment by tassosblackg** [comment]
Did you installed the specific branch or did you run `make`, `make install` in the main branch?  Because if I `git checkout ewlsh/gnome-40 `  the above branch and run `make`, it gives me an error 

> sassc --omit-map-comment _stylesheet.scss stylesheet.css
> make: sassc: No such file or directory
> make: *** [Makefile:66: stylesheet.css] Error 127
